recipes/pcl/all/conanfile.py

```diff
@@ -0,0 +1,242 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.system import package_manager
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class PclConan(ConanFile):
+    name = "pcl"
+    description = "Point Cloud Library"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/PointCloudLibrary/pcl"
+    topics = ("computer-vision", "point-cloud", "pcd", "pointcloud")
```

Comment:
```suggestion
    topics = ("computer vision", "point cloud", "pointcloud", "3d", "pcd", "ply", "stl", "ifs", "vtk")
```
I the rest of the io format tags are worth including to make the package easier to find based on these. For example, there's an rply package for .ply in ConanCenter and PCL can be used as an alternative for it.
Also, I think the hyphen is not necessary.